js/binancetr.js

```diff
@@ -201,6 +216,49 @@ module.exports = class binancetr extends Exchange {
         return result;
     }
 
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'symbol': this.marketId (symbol),
+        };
+        if (limit === undefined) {
+            limit = 100; // default = 100, max 5000. Valid values are 5, 10, 20, 50 100 and 500
```

Comment:
I think we do not need to set the value of the limit if it is not defined by the user since it is not a mandatory parameter

Reply:
instead I would make the error the first if clause and the else if clause to be `else if (limit !== undefined) { request[limit] = limit; } `

Reply:
Ok. Done.
Changed in the next commit to this:
```
        if ((limit !== 5) && (limit !== 10) && (limit !== 20) && (limit !== 50) && (limit !== 100) && (limit !== 500)) {
            throw new BadRequest (this.id + ' fetchOrderBook() limit argument must be undefined, 5, 10, 20, 50, 100 or 500, default is 100');
        } else if (limit !== undefined) {
             request['limit'] = limit; 
        } 
'''